apis/pubsub/v1alpha1/subscription_types.go

```diff
@@ -87,6 +87,10 @@ type SubscriptionParameters struct {
 	// +optional
 	PushConfig *PushConfig `json:"pushConfig,omitempty"`
 
+	// BigqueryConfig is a parameter which configures bigquery delivery.
+	// +optional
+	BigqueryConfig *BigqueryConfig `json:"bigqueryConfig,omitempty"`
```

Comment:
> I had to rename BigQueryConfig -> BigqueryConfig

Got it. But I think we can keep the Go field as `BigQueryConfig` and just change the `json` tag. What do you think?

```suggestion
	// BigQueryConfig is a parameter which configures BigQuery delivery.
	// +optional
	BigQueryConfig *BigQueryConfig `json:"bigqueryConfig,omitempty"`
```

Reply:
Sounds great, I will try.